samples/MvcSample.Web/HomeController.cs

```diff
@@ -113,6 +113,22 @@ public ActionResult SaveUser(User user)
         }
 
         /// <summary>
+        /// Action that shows binding of form values to <see cref="IFormCollection"/>.
+        /// </summary>
+        public ActionResult PostForm(IFormCollection form)
+        {
+            if (!ModelState.IsValid)
```

Comment:
what are the conditions where `form` might be null?

This is kinda a wierd sample. Is the intent to distinguish between a get and a post? Normally you'd do that with `[HttpPost]`